modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -203,9 +207,9 @@ sub core_pipeline_analyses {
             -module          => 'Bio::EnsEMBL::Compara::RunnableDB::HomologyAnnotation::SpeciesFactory',
             -max_retry_count => 1,
             -input_ids       => [{
-                'registry_file'      => $self->o('reg_conf'),
-                'species_list'       => $self->o('species_list'),
-                'species_list_file'  => $self->o('species_list_file'),
+                'registry_file'      => $self->o('registry_file'),
+                'species_list'       => $self->o('species'),
+                'species_list_file'  => $self->o('species_list'),
```

Comment:
Smart! I would have changed our runnable names as well, but it is clearly not necessary :facepalm: